tests/integration/docker/info.bats

```diff
@@ -31,6 +31,6 @@ setup() {
 @test "Container info" {
 	container=$(random_name)
 	$DOCKER_EXE run -itd --name $container busybox
-	$DOCKER_EXE info| grep "Containers: 1"
+	$DOCKER_EXE info| grep "^ Running: 1"
```

Comment:
If don't check anything if we just check for a number. We can actually do much better, just ask docker for the status of $container:

```
$ docker inspect --type container --format '{{ .State.Status }}' $container
running
```